eng/Versions.props

```diff
@@ -186,6 +186,11 @@
     <MicrosoftBuildUtilitiesCoreVersion>16.9.0</MicrosoftBuildUtilitiesCoreVersion>
     <MicrosoftBuildLocatorVersion>1.2.6</MicrosoftBuildLocatorVersion>
     <MicrosoftBuildUtilitiesCoreVersion>16.9.0</MicrosoftBuildUtilitiesCoreVersion>
+    <!--
+      Temporarily override the Microsoft.NET.Test.Sdk version Arcade defaults to. That's incompatible w/ test
+      framework in current .NET SDKs.
```

Comment:
Is there some issue tracking fixing this?

Reply:
I think we should probably merge this to get the helix matrix passing again, but we should probably open some issue to revert this so we don't forget about it.

Reply:
The issue should be something for @MarcoRossignoli to manage.

@MarcoRossignoli, this removal in #38506 didn't work ☹️ I guess that means your SDK insertions haven't corrected the incompatibility w/ older test hosts.

Reply:
I'll take a look at it today

Reply:
Looks like something transient, for some reason `testhost.dll` file that is present during test startup (test platform locates testhost.dll from nuget or local directory) is missing after some time.
Here my test with the analysis https://github.com/dotnet/aspnetcore/pull/38937